### PR TITLE
Bugfix/text starting with numbers

### PR DESCRIPTION
**Pull request by McCourtRC**

### CODE REVIEW

File: lib/tests/convert.test.js [hunk 1]
```diff
@@ -4,14 +4,15 @@ const { fromSrt } = require('subtitles-parser');
 const convert = require('../converters');
 const badTextMultipleReturns = require('./mocks/badTextMultipleReturns');
 const badIndexSequence = require('./mocks/badIndexSequence');
-const badIndexNotNumerical = require('./mocks/badIndexNotNumerical');
+// const badIndexNotNumerical = require('./mocks/badIndexNotNumerical');
```
Comment (snooplyin) [inline lib/tests/convert.test.js]: remove?

File: lib/tests/convert.test.js [hunk 2]
```diff
@@ -34,10 +34,11 @@ describe('#convert', () => {
     expect(parsed[1].text).to.equal('Can’t see original dreams');
   });
 
-  it('should catch non numerical indices', () => {
-    const { status } = convert(badIndexNotNumerical, '.srt', '.srt');
-    expect(status.invalidEntries).to.not.be.empty;
-  });
+  // TODO!
```
Comment (snooplyin) [inline lib/tests/convert.test.js]: Is this TODO for later or no?

Comment (McCourtRC) [reply lib/tests/convert.test.js]: I would like to look into this in the future.  I can delete and make a note/ticket for it, or leave here as a reminder.  What works best?

Comment (McCourtRC) [reply lib/tests/convert.test.js]: Removed the unused test and created a ticket to investigate later.

**Pull request merged**